masterlist.yaml

```
@@ -22981,17 +22981,24 @@ plugins:
         nav: 4
 
   - name: 'WindhelmEntranceOverhaul.esp'
-    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/19400/' ]
+    url: 
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/19400/'
+        name: 'City Entrances Overhaul - Windhelm'
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/65092/'
+        name: 'Rob''s Bug Fixes - City Entrances Overhaul - Windhelm'
+    msg:
+      - <<: *patchUpdateAvailable
+        subs: [ '[Rob''s Bug Fixes - City Entrances Overhaul - Windhelm](https://www.nexusmods.com/skyrimspecialedition/mods/65092/)' ]
+        condition: 'checksum("WindhelmEntranceOverhaul.esp", A91FF679)'
     dirty:
       # version: 1.3
       - <<: *quickClean
         crc: 0xA91FF679
         util: '[SSEEdit v4.0.2](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
         itm: 13
```

`0x6972E572` is actually for the Banners Fix file; it's also a replacer plugin. Judging by the author's advice & the timestamps of the two plugins, it'd make more sense to keep that one & remove this one (along with the version comment) as it appears to be the more up-to-date of the two.

```suggestion
      - <<: *quickClean
        crc: 0x6972E572
        util: '[SSEEdit v4.0.3](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
        itm: 13
```